src/components/SubscriptAvatar.js

```diff
@@ -47,45 +51,60 @@ function SubscriptAvatar(props) {
     if (props.noMargin) {
         marginStyle = {};
     }
+
+    const mainAvatar = (
```

Comment:
Question, can be `MainAvatar` and below use it as
```
<MainAvatar />
```
I feel it odd seeing it like that 
same for `secondaryAvatar`
Also maybe creating them outside of the component (?)

Reply:
Yeah, I also find it a little bit odd, but I'm just following the patterns that already exist.

Ideally I would create outside a wrapper component that changes the container conditionally, but I think it's better to stick with the existing patterns in this case